src/main/java/creativei/service/impl/FollowUpServiceImpl.java

```diff
@@ -22,6 +22,16 @@ public FollowUp createFollowUp(FollowUp followUp) throws InvalidParamRequest {
 
         return followUpDao.save(followUp);
     }
+
+    @Override
+    public FollowUp updateFollowUp(FollowUp followUp) throws InvalidParamRequest {
+        if (!validateFollowUp(followUp))
+            throw new InvalidParamRequest("Required field can not be empty");
```

Comment:
logger.error here

Reply:
sir when this exception is catch by the method in  manager ,that method include the logger,over there.